### ISSUE TITLE
Setting value for property in event listener results in DBALException

### ISSUE THREAD

**Issue by UB3RL33T** [MetaModels/contao-frontend-editing#40]
## Checklist before I submit this issue report

I confirm that:
- [x] I have tested this with the latest version available
- [x] I have read documentation @ http://metamodels.readthedocs.org/en/ or http://metamodels.readthedocs.org/de/
- [x] I have checked the Contao community forums for references https://community.contao.org/
- [x] I have checked existing issues for duplicates and found none @ https://github.com/MetaModels/contao-frontend-editing/issues?q=is%3Aissue

## My environment is:

(Please fill in the actual values from your environment)

| Key                                 | Value    | Comments                         |
| ----------------------------------- | ---------| ---------------------------------|
| PHP version:                        |    7.2      |                                  |
| Contao version:                     |     4.9     |                                  |
| MetaModels version:                 |    2.2.x-dev      |                                  |
| Installation via composer:          | yes |                                  |
| Installed MetaModels packages:      |          |                                  |
| DCG version:                        |    2.2.x-dev      |                                  |

## Issue description

In my current setup I have a Metamodel which references `tl_member` so each metamodel is "owned" by a frontend user.
The referencing attribute is a select attribute since the Admin may want to change ownership of the Metamodel.

I'm using an event listener on https://github.com/contao-community-alliance/dc-general/blob/master/src/Event/PrePersistModelEvent.php to set this property on save for the currently logged in user, like so:

`$model->setProperty('member_id', $user->id)`

This worked fine until the latest update, now I'm getting a DBALException:

```
An exception occurred while executing 'SELECT * FROM tl_member t WHERE t.id=:id LIMIT 1' with params ["4"]:

Value for :id not found in params array. Params array key should be "id"
```
I'm not sure if the issue lies within https://github.com/MetaModels/attribute_select/blob/master/src/Attribute/Select.php#L100, otherwise it would probably be an Issue in Doctrine?

The main Issue is that I can't roll back the changes because the checksum verification for the package fails for previous `composer.lock` versions

```
$ composer install
...
  - Installing metamodels/attribute_alias (dev-feature/2.2.0): Downloading (100%)         
                                                                                                                                                                       
  [UnexpectedValueException]                                                                                                                                           
  The checksum verification of the file failed (downloaded from https://packages.cyberspectrum.de/p/Eg...B6)  
                                                                                                                                                                       
```

## Steps to reproduce

1. Setup a Metamodel with FEE
2. Add a select property which references `tl_member`
3. Setup an event listener on the PrePersistModelEvent which sets the referencing property (member_id) to the current logged in user id
4. Go to the Metamodel in the Frontend and create a new Metamodel (save)

## Describe the behaviour of the application

Upon saving the new Metamodel, the application throws a DBALException.

## Describe the expected behaviour of the application

Upon saving the new Metamodel should be created and the property should be set (and saved)

## Screenshots
The resulting error
![image](https://user-images.githubusercontent.com/21123031/93873226-a8e27800-fcd1-11ea-985e-254a99359f5d.png)

**Comment by zonky2** [comment]
which version of DBAL?

see https://github.com/MetaModels/core/issues/1425

**Comment by amenk** [comment]
Hi @zonky2  we seem to be already / still on 2.10.4 and not on 2.11
I attached the output of `composer show`
[composer.show.txt](https://github.com/MetaModels/contao-frontend-editing/files/5261075/composer.show.txt)

**Comment by zonky2** [comment]
try in Select.php

```diff
    public function widgetToValue($varValue, $itemId)
    {
        if (null === $varValue) {
            return null;
        }
        // Lookup the value.
        $value = $this->connection->createQueryBuilder()
            ->select('*')
            ->from($this->getSelectSource(), 't')
            ->where('t.' . $this->getIdColumn() . '=:id')
-           ->setParameter('t.id', $varValue)
+           ->setParameter('id', $varValue)
            ->setMaxResults(1)
            ->execute()
            ->fetch(\PDO::FETCH_ASSOC);

        return $value;
    }
```

**Comment by UB3RL33T** [comment]
I made the edit in my local vendor and tried it, this'll work :)

**Comment by zonky2** [comment]
fixed in MM 2.2
